src/stories/ConsoleIcon.stories.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { ConsoleIcon } from "../common/components/icons/ConsoleIcon";
+import { Story, Meta } from "@storybook/react/types-6-0";
+
+export default {
+  title: "ConsoleIcon",
```

Comment:
Maybe would make sense to group all the menu icons under one story?

Reply:
Done.